### DISCUSSION TITLE
Action to select last used tab/window

### DISCUSSION THREAD

**Discussion by sam701** [kovidgoyal/kitty#4943] [Q&A] [answered]
Is it possible to have a shortcut to select last used tab and/or window? I would like to have `ctrl-tab` mapped to `last_used_tab` instead of previous.

**Comment by page-down** [comment, marked as answer]
Do you want to switch between the last active tabs/windows?
You can use `goto_tab`, `nth_window` with negative numbers.

https://sw.kovidgoyal.net/kitty/conf/#tab-management
https://sw.kovidgoyal.net/kitty/actions/#action-goto_tab

https://sw.kovidgoyal.net/kitty/overview/#windows
https://sw.kovidgoyal.net/kitty/actions/#action-nth_window

**Comment by kovidgoyal** [comment]
goto_tab -1

**Comment by sam701** [comment]
Thank you, works great :+1:

 Is it possible to go to the second previously used tab by holding the `ctrl` and pressing the tab twice? This is a common behavior in some other tools, e.g. iTerm, firefox.

**Reply by poige** [reply]
@kovidgoyal that one is useful when dealing with lots of tabs

**Comment by haydonryan** [comment]
Thanks for posting this. Very thankful for this being implemented.

I wonder if it would be worth adding to the code  `last_used_tab ` instead of `-1 or any other number lager than the highest tab number`.  Generally sentinel values aren't as descriptive in a configuration especially for those who are new.

**Reply by kovidgoyal** [reply]
It's not a sentinel value, that was a holdover from olden times. https://github.com/kovidgoyal/kitty/commit/75f5908009a16752493015b1e9b39d778f443639

**Comment by haydonryan** [comment]
i was wondering why i wasn't getting the behavior i expected, then  found a post that said to use `goto tab 0` instead the true previous. eg switch there and back.
